android-gradle-plugin-aspectj/bintray.gradle

```diff
@@ -41,18 +41,18 @@ if (bintrayUser && bintrayApiKey) {
             issueTrackerUrl = issuesUrl
             vcsUrl = gitUrl
             licenses = [allLicenses]
-            labels = ['android', 'gradle']
+            labels = ['android', 'gradle', 'java', 'aspectj']
             publicDownloadNumbers = false
 
             githubRepo = githubRepository //Optional Github repository
-            githubReleaseNotesFile = 'README.md' //Optional Github readme file
+            githubReleaseNotesFile = 'CHANGELOG.md' //Optional Github readme file
```

Comment:
I'm not seeing this file. Is this file going to be added later?

Reply:
Yes, this will come in when the linked documentation PR comes in. (see comment above)